Assets/VRTK/Scripts/Controls/3D/VRTK_Spring_Lever.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Lever|Controls3D|0071
+using UnityEngine;
+
+namespace VRTK {
+
+    /// <summary>
+    /// This script extends VRTK_Lever to add spring force toward whichever end of the lever's range it is closest to.
+    /// </summary>
+    /// <remarks>
+    /// The script will instantiate the required Rigidbody, Interactable and HingeJoint components automatically in case they do not exist yet. The joint is very tricky to setup automatically though and will only work in straight forward cases. If there are any issues, then create the HingeJoint component manually and configure it as needed.
+    /// </remarks>
+    public class VRTK_Spring_Lever : VRTK_Lever {
+
+        public float springStrength = 10;
```

Comment:
this will need a `[Tooltip("")]` above it to provide an inspector tooltip (and will also be used to generate the docs from it)